include/DREAM/Settings/OptionConstants.enum.hpp

```diff
@@ -63,7 +63,8 @@ enum ion_neutral_advection_mode {
 
 enum ion_source_type {
 	ION_SOURCE_NONE=1,
-	ION_SOURCE_PRESCRIBED=2
+	ION_SOURCE_PRESCRIBED=2,
+	ION_SOURCE_PRESCRIBED_CONSTANT=3
```

Comment:
What does the ``CONSTANT`` refer to here? I think it would be better to call it ``ION_SOURCE_PRESCRIBED_VOLUMETRIC`` or so, since it shouldn't be constant in time nor space.